frontend/src/metabase/admin/settings/components/SiteUrlDescription.jsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { t } from "ttag";
+
+const STRONG_TEXT = "Only change this if you know what you're doing!";
+
+function SiteUrlDescription({ description }) {
+  if (!description.includes(STRONG_TEXT)) {
+    return description;
+  }
+  // eslint-disable-next-line no-unused-vars
+  let [_, text] = description.split(STRONG_TEXT);
+  text = text.trim();
+  return (
+    <React.Fragment>
+      <strong>{t(STRONG_TEXT)}</strong> {t(text)}
+    </React.Fragment>
+  );
+}
```

Comment:
The "Only change this if you know what you're doing!" part of the copy is bold on the mockup.

Settings descriptions are served from the backend, so making part of the string bold is a bit tricky, had to introduce the `descriptionComponent` prop for that.

Please LMK if this can be simplified or done more cleanly ✌️

Reply:
IMO it's not worth jumping through hoops to ensure the entire message comes from the backend. Instead, we could delete the "Only change this" sentence from the backend string and replace `descriptionComponent` with some prop containing text for a bolded prefix. e.g.

``` warningMessage: t`Only change this if you know what you're doing!` ```

That property still might not be defined for any other setting but it feels more general than a component that searches for that string specifically. 

I also worry that the strings wouldn't get picked up correctly for translation if we split and recombine them this way.

Reply:
@paulrosenzweig good call! Thank you